### ISSUE TITLE
Feature request: histogram as plot type for `clusterplot`

### ISSUE THREAD

**Issue by marabuuu** [haesleinhuepf/stackview#56]
Hi @haesleinhuepf,

It is really cool to have the napari-clusters-plotter functionality in jupyter notebooks via the `clusterplot` function. 
It would be cool to be able to choose other plot types, like histograms (e.g. for intensity measurements).

Best,
Mara

**Comment by haesleinhuepf** [comment]
Yes, we could do it like in napari: if one chooses the same axis in x and y, we show a histogram instead of a scatter plot.
